internal/command/jsonplan/plan.go

```diff
@@ -39,6 +39,9 @@ const (
 	ResourceInstanceDeleteBecauseNoMoveTarget     = "delete_because_no_move_target"
 	ResourceInstanceReadBecauseConfigUnknown      = "read_because_config_unknown"
 	ResourceInstanceReadBecauseDependencyPending  = "read_because_dependency_pending"
+
+	ManagedResourceMode = "resource"
```

Comment:
I think this should be "managed", unless I'm missing something.

Reply:
Yes, fixed.